docs/versionhistory.rst

```diff
@@ -8,6 +8,7 @@ This library adheres to `Semantic Versioning <http://semver.org/>`_.
 **UNRELEASED**
 
 - Fixed compilation of C extension failing on GCC 14
+- Fixed compiler warnings when building C extension
```

Comment:
Does this warrant a new changelog entry? Weren't these two PRs actually both to fix the compilation problem on GCC 14?

Reply:
No, this PR fixes unrelated compiler warnings. The other own fixed an actual build error.